randovania/games/am2r/logic_database/The Tower.txt

```diff
@@ -929,20 +929,27 @@ Extra - minimap_data: [{'x': 33, 'y': 34}, {'x': 33, 'y': 35}, {'x': 33, 'y': 36
   > Event - Tester
       All of the following:
           Any of the following:
-              # Energy requirements
+              # Weapon Requirements
+              Missiles or Plasma Beam or Power Bombs or Super Missiles
+              Combat (Advanced) and Can Use Charged Bomb Jump
+              Combat (Hypermode) and Can Use Bombs
+              Combat (Expert) and Damage Boost (Advanced) and Knowledge (Advanced) and Normal Damage ≥ 299
```

Comment:
could use a comment that you can circumvent the outer shell by damage boosting into it and shooting it from the inside